liquidcore/home/signals.py

```diff
@@ -19,3 +24,28 @@ def kill_user_sessions(user, **kwargs):
 def check_user_username(sender, instance, **kwargs):
     assert all(x in USERNAME_CHARS for x in instance.username), \
         f'Bad username: "{instance.username}"'
+
+
+@receiver(post_save, sender=User)
+def create_user_everywhere(sender, instance, created=None, **kwargs):
+    if created:
+        payload = {
+            "Meta": {
+                "USERNAME": instance.username
+            }
+        }
+        nomad_ip = os.getenv('SERVICE_ADDRESS')
+        if nomad_ip:
+            res = requests.post(
+                f'http://{nomad_ip}:4646/v1/job/liquid-createuser/dispatch',
```

Comment:
Let's have a different, specific env called `NOMAD_URL` that is `http://IP:PORT` that you fill out from the nomad template. I want this kind of breakage to be obvious from configuration when we migrate away from nomad.